h5netcdf/core.py

```diff
@@ -1127,6 +1211,7 @@ def __init__(self, path, mode="r", invalid_netcdf=False, phony_dims=None, **kwar
         track_order = kwargs.pop("track_order", track_order_default)
 
         self.decode_vlen_strings = kwargs.pop("decode_vlen_strings", None)
+        self._auto_complex = kwargs.pop("auto_complex", None)
```

Comment:
Why not use real keyword arguments here (which could be documented in the docstring) instead of putting them in `**kwargs`?

Reply:
I was hesitant to do this. But it will surely be more visible, thanks!

Reply:
Not needed with latest approach.